src/lvm.c

```diff
@@ -1161,11 +1180,21 @@ void luaV_execute (lua_State *L, CallInfo *ci) {
     ci->u.l.trap = 1;  /* assume trap is on, for now */
   }
   base = ci->func + 1;
+#ifdef WITH_YK
+  YkMT *mt = yk_mt_new();
+  yk_mt_hot_threshold_set(mt, 5); /* YKFIXE: allow changing threshold */
```

Comment:
s/YKFIXE/YKFIXME

Reply:
Nice. Fixed in 76192d650

Reply:
You somehow made it worse. :-D

Reply:
It was a test. You passed.

Fixed in [276f0e2](https://github.com/ykjit/yklua/pull/2/commits/276f0e286001884fa68920cdc17806757abeb180)